### PR TITLE
1824: implement more part 2

### PR DESCRIPTION
**Pull request by perwestling**
This PR contains 4 commits:
1. Export a normal train at the end of any OR set
2. Use status_str to show information instead of using base type abilities, plus show type there
3. Support simplified logos for regional and staatsbahn
4. Improve log text for coal/pre-staatsbahn dividend

### CODE REVIEW

**Comment by perwestling** [comment]
Standard logos looks like this:
image

Simplified logos:
image

File: lib/engine/game/g_1824.rb [hunk 1]
```diff
@@ -307,12 +327,33 @@ def setup
         end
       end
 
+      def timeline
+        @timeline ||= ['At the end of each OR set, the cheapest train in bank is exported.'].freeze
+        @timeline
```
Comment (tobymao) [inline lib/engine/game/g_1824.rb:332]: extra timeline is not necessary

**Comment by tobymao** [comment]
compact your svgs

File: lib/engine/game/g_1824.rb (hunk 1 above)
Comment (perwestling) [reply lib/engine/game/g_1824.rb]: Fixed

**Comment by perwestling** [comment]
> compact your svgs

Fixed.
Should I compact all the other svgs of 1824 as well?

**Pull request merged**